pubspec.yaml

```diff
@@ -25,12 +25,12 @@ dependencies:
   #cloud_firestore: ^3.1.5 TODO: turn back on once needed
   hive: ^2.0.5
   hive_flutter: ^1.1.0
-  expansion_tile_card: ^2.0.0
   flutter:
     sdk: flutter
   get:
   line_icons: ^2.0.1
   cupertino_icons: ^1.0.2
+  url_launcher: ^6.0.20
```

Comment:
Do we need this?

Reply:
Yes, this one is used to activate www.totu.sds.pl link